.github/workflows/tests.yml

```diff
@@ -0,0 +1,40 @@
+name: Backend Tests CI
+
+on:
+  push:
+    branches:
```

Comment:
In the acceptance criteria the card requires that CI/CD test pipeline should be triggered each time there is a push to any branch, but when the task is explained it requires tests each push to main/dev branch. We should check requirements.